tests/parser_tests/translate_scripts_test.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+from mentat.parsers.block_parser import BlockParser
+from mentat.parsers.git_parser import GitParser
+from mentat.parsers.replacement_parser import ReplacementParser
+from scripts.translate_transcript import translate_message
+
+
+@pytest.fixture
+def block_format():
+    with open("format_examples/block.txt") as f:
+        return f.read()
+
+
+@pytest.fixture
+def replacement_format():
+    with open("format_examples/replacement.txt") as f:
+        return f.read()
+
+
+@pytest.fixture
+def git_diff_format():
+    with open("format_examples/git_diff.txt") as f:
+        return f.read()
+
+
+# This one doesn't pass. I think it's actually a bug with the parser as if I print out the final parsed
```

Comment:
@PCSwingle I'll look into this tomorrow but I wanted to bring your attention to it because I thought you might just know what the problem is.

Reply:
I'm looking into it. You can also mark tests like this with @pytest.mark.xfail to mark it as failing but still let everything pass

Reply:
Holy cow I found the bug and it's actually a massive bug; I can't believe this isn't caught by any of our tests. I'll push a commit soon that fixes it (super easy fix) and will make it obvious what the bug was haha. Thanks for finding this!!